frappe/custom/doctype/customize_form/customize_form.py

```diff
@@ -59,6 +59,8 @@
 	'report_hide': 'Check',
 	'allow_on_submit': 'Check',
 	'translatable': 'Check',
+	'mandatory_depends_on': 'Data',
```

Comment:
Conditions defined in these fields through customize from is not persistent. Property setter should be created for conditions defined through customize form.